lightbluetent/templates/rooms/manage.html

```diff
@@ -256,39 +257,47 @@ <h1><a href={{ url_for("rooms.new_password", room_id=room.id) }}
                         <div class="input-group-prepend">
                             <label class="input-group-text">Repeat every</label>
                         </div>
-                        <select class="custom-select" id="recurrence_select">
-                            <option selected>Choose...</option>
-                            <option value="day">day</option>
-                            <option value="weekday">weekday</option>
-                            <option value="week">week</option>
-                            <option value="month">month</option>
-                            <option value="year">year</option>
+                        <select class="custom-select {% if errors.frequency is defined %} is-invalid{% endif %}"
+                                id="recurrence_select" name="frequency">
+                            <option {% if frequency is not defined %}selected{% endif %}>Choose...</option>
```

Comment:
Can we use the enum here? either pass it in to the template, or set it as a global in app.py